packages/react-components/react-combobox/src/components/Combobox/useCombobox.tsx

```diff
@@ -174,15 +175,14 @@ export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLIn
       },
     }),
     input: triggerSlot,
-    listbox: listboxSlot,
+    listbox: open || hasFocus ? listboxSlot : undefined,
```

Comment:
nit: ideally you could do this check before even calling `resolveShorthand` above, and save doing some unnecessary work. But if `useComboboxPopup` and/or `useTriggerListboxSlots` don't work without the listbox, then it's fine to leave it here.